test/unit/strategy.test.ts

```diff
@@ -120,16 +122,16 @@ describe('Deployed Strategy', () => {
     shares: 3,
     porterUri: 'https://_this.should.crash',
   };
-  const aliceSecretKey = SecretKey.fromBytes(aliceSecretKeyBytes);
-  const bobSecretKey = SecretKey.fromBytes(bobSecretKeyBytes);
+  const aliceSecretKey = SecretKey.fromBEBytes(aliceSecretKeyBytes);
+  const bobSecretKey = SecretKey.fromBEBytes(bobSecretKeyBytes);
 
   afterEach(() => {
-    jest.clearAllMocks();
+    jest.restoreAllMocks();
```

Comment:
This was the culprit. See [this excellent](https://stackoverflow.com/questions/58151010/difference-between-resetallmocks-resetmodules-resetmoduleregistry-restoreallm) StackOverflow answer.